perllib/FixMyStreet/App/Controller/Admin/Triage.pm

```diff
@@ -0,0 +1,108 @@
+    my ( $self, $c ) = @_;
+
+    $c->stash->{body} = $c->forward('body_find', [ $c->cobrand->council_area ]);
+    $c->forward( 'stash_report_filter_status' );
+    $c->forward( 'load_and_group_problems' );
+    $c->stash->{page} = 'reports'; # So the map knows to make clickable pins
+
+    if ($c->get_param('ajax')) {
+        my $ajax_template = $c->stash->{ajax_template} || 'reports/_problem-list.html';
+        $c->detach('ajax', [ $ajax_template ]);
+    }
+
+    my @categories = $c->stash->{body}->contacts->not_deleted->search( undef, {
+        columns => [ 'id', 'category', 'extra' ],
+        distinct => 1,
+        order_by => [ 'category' ],
+    } )->all;
+    $c->stash->{filter_categories} = \@categories;
+    $c->stash->{filter_category} = { map { $_ => 1 } $c->get_param_list('filter_category', 1) };
+    my $pins = $c->stash->{pins} || [];
+
+    my %map_params = (
+        latitude  => @$pins ? $pins->[0]{latitude} : 0,
+        longitude => @$pins ? $pins->[0]{longitude} : 0,
+        area      => [ $c->stash->{wards} ? map { $_->{id} } @{$c->stash->{wards}} : keys %{$c->stash->{body}->areas} ],
+        any_zoom  => 1,
+    );
+    FixMyStreet::Map::display_map(
+        $c, %map_params, pins => $pins,
+    );
+}
+
+sub stash_report_filter_status : Private {
+    my ( $self, $c ) = @_;
+    $c->stash->{filter_problem_states} = { 'for triage' => 1 };
+    return 1;
+}
+
+sub setup_categories : Private {
+    my ( $self, $c ) = @_;
+
+    if ( $c->stash->{problem}->state eq 'for triage' ) {
+        $c->stash->{holding_options} = [ grep { $_->send_method && $_->send_method eq 'Triage' } @{$c->stash->{category_options}} ];
+        $c->stash->{holding_categories} = { map { $_->category => 1 } grep { $_->send_method && $_->send_method eq 'Triage' } @{$c->stash->{category_options}} };
+        $c->stash->{end_options} = [ grep { !$_->send_method || $_->send_method ne 'Triage' } @{$c->stash->{category_options}} ];
+        $c->stash->{end_categories} = { map { $_->category => 1 } grep { !$_->send_method || $_->send_method ne 'Triage' } @{$c->stash->{category_options}} };
+        delete $c->stash->{categories_hash};
+        my %category_groups = ();
+        for my $category (@{$c->stash->{end_options}}) {
+            my $group = $category->{group} // $category->get_extra_metadata('group') // [''];
+            # this could be an array ref or a string
+            my @groups = ref $group eq 'ARRAY' ? @$group : ($group);
+            push( @{$category_groups{$_}}, $category ) for @groups;
+        }
+        my @category_groups = ();
+        for my $group ( grep { $_ ne _('Other') } sort keys %category_groups ) {
+            push @category_groups, { name => $group, categories => $category_groups{$group} };
+        }
+        $c->stash->{end_groups} = \@category_groups;
+    }
+
+    return 1;
+}
+
+sub update : Private {
+    my ($self, $c) = @_;
+
+    my $current_category = $c->get_param('holding_category');
```

Comment:
This is a dropdown in the interface, is read here, but nothing can be done with it - I can't change it to a different holding category. Should the front end be displaying this rather than letting it be edited?